.vscode/launch.json

```diff
@@ -36,5 +36,20 @@
             ],
             "preLaunchTask": "build-wzdx-translator",
         },
+        {

```

Comment:
Looks like we've removed the functions framework references from the code, do we need to remove the other launch task or will it still work?

Reply:
The other two launch tasks weren't running correctly - those have been removed and the existing flask launch task has been updated to launch correctly